hudi-flink/src/main/java/org/apache/hudi/source/StreamReadOperator.java

```diff
@@ -64,7 +64,7 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(StreamReadOperator.class);
 
-  private static final int MINI_BATCH_SIZE = 1000;
+  private static final int MINI_BATCH_SIZE = 2048;
 
```

Comment:
I see your comment:
> ParquetColumnarRowSplitReader#batchSize is 2048, so Changing MINI_BATCH_SIZE to 2048 will reduce memory cache

Thanks, do we have some metrics to illustrate the benefit.

Reply:
now i dont have any metrics, but from the code，the time that data is stroed in the memory should be reduce to avoid entering the Old Generation.

https://github.com/apache/hudi/blob/b87e95d621e9fd2d1aebca4c49f599b6437aaf83/hudi-flink/src/main/java/org/apache/hudi/table/format/cow/ParquetColumnarRowSplitReader.java#L306

Reply:
@danny0405  from memory dump：
![image](https://user-images.githubusercontent.com/36881649/155911679-39fa8ccc-edb5-4425-996f-25e76aaf6022.png)